internal/serverinstall/nomad.go

```diff
@@ -1131,6 +1211,50 @@ func (i *NomadInstaller) UpgradeFlags(set *flag.Set) {
 		Target: &i.config.hostVolume,
 		Usage:  "Nomad host volume name.",
 	})
+
+	set.BoolVar(&flag.BoolVar{
+		Name:    "nomad-consul-service",
+		Target:  &i.config.consulService,
+		Usage:   "Create service for Waypoint UI in Consul.",
+		Default: true,
+	})
```

Comment:
Does our flag system work with `true` by default bools?

Reply:
It does! And `-nomad-consul-service=false` also works for turning it off (note that this flag was introduced prior to this PR)

Reply:
Excellent!